recipe_roulette.py

```diff
@@ -42,9 +44,9 @@ def get_user_input():
                   .format(user_ingredient.lower()))
             break
         elif user_input.lower() == 'n':
-            user_ingredient = get_random_ingredient()
+            user_ingredient = get_random_ingredient() # this is a bit unneccesarry, can very easily inline this function's logic 

```

Comment:
Could you please specify what this comment is referring too?: "this is a bit unneccesarry, can very easily inline this function's logic "

Reply:
```user_ingredient = random.choice(default_ingredient_list)```  instead of ```user_ingredient = get_random_ingredient()```

The function is only called once in the code and can be simplified to a single line so I really struggle to see why it should be a function at all.